get_env_data.py

```diff
@@ -11,7 +11,7 @@
 plex_needed = utils.input_yesno("-- Plex --\nAre you logged into this server with a Plex account?")
 if plex_needed:
     username = input("Please enter your Plex username: ")
-    password = input("Please enter your Plex password: ")
+    password = utils.input_hidden("Please enter your Plex password: ")
```

Comment:
As said before, it should catch wrong credentials and ask again.

Reply:
@twolaw I have not made any changes since the last review.

I only rebased against the current master, maybe you go notification on the rebase push.